packages/steiger/src/app.ts

```diff
@@ -1,25 +1,26 @@
-import { createEffect, sample } from 'effector'
+import { createEffect, sample, combine } from 'effector'
 import { debounce, not } from 'patronum'
-import fsdRules from '@feature-sliced/steiger-plugin'
 import type { Rule, Folder } from '@steiger/types'
 import type { AugmentedDiagnostic } from '@steiger/pretty-reporter'
 
 import { scan, createWatcher } from './features/transfer-fs-to-vfs'
 import { defer } from './shared/defer'
-import { $config, type Config } from './models/config'
+import { $config, $rules } from './models/config'
 
 function getRuleDescriptionUrl(ruleName: string) {
   return new URL(`https://github.com/feature-sliced/steiger/tree/master/packages/steiger-plugin-fsd/src/${ruleName}`)
 }
 
-const enabledRules = $config.map((config) => {
+type Config = typeof $config
+
+const $enabledRules = combine($config, $rules, (config, rules) => {
   const ruleConfigs = config?.rules
 
   if (ruleConfigs === undefined) {
-    return fsdRules
+    return rules
   }
 
-  return fsdRules.filter(
+  return rules.filter(
     (rule) => !(rule.name in ruleConfigs) || ruleConfigs[rule.name as keyof typeof ruleConfigs] !== 'off',
   )
 })
```

Comment:
question: why do we need `$rules` here? I thought they basically contain the same things, only `$config` contains more information

Reply:
@illright 
`$rules` contains all rule definitions (registered by plugins) the tool can run. `$config` contains configurations like rule-to-severity map (in the future "files" and "ignore" globs, and other settings). So, based on that info we determine rule instances that need to be run (enabledRules)